package.json

```
@@ -46,7 +46,8 @@
   },
   "peerDependencies": {
     "lexical": "^0.11.0",
-    "vue": "^3.2.0"
+    "vue": "^3.2.0",
+    "yjs": ">=13.5.22"
```

Why is this listed like this rather than standard?
```suggestion
    "yjs": "^13.5.22"
```

If you change, be sure to reset `pnpm-lock.yaml` too